src/test.ts

```diff
@@ -1,17 +1,16 @@
-import {Address, LambdaInputEvent, handler} from './index';
-import {logger} from "./logger";
-
-(async function() {
-  const address: Address = {
-      "city": "München",
-      "country": "Germany",
-      "zip": "81541",
-      "street": "Zugspitzstrasse",
-      "hn": "19"
-  };
-  const requestObject: LambdaInputEvent = {
-    body: JSON.stringify(address)
-  };
-  const result = await handler(requestObject);
-  logger.info(JSON.stringify(result))
-})();
+import { Address, handler, LambdaInputEvent } from './index'
+import { logger } from './logger'
+;(async function () {
```

Comment:
There is no use for this ; is it?

Apart from that I approve. Have you found a  way to import this linting settings into intellij auto formatting?

Reply:
This felt strange for me, too. But I think it's because we left the semicolon from the import above, and the self invoking function might interfere with that.

Reply:
Tried it without, also works.

Reply:
If you run the linter again, it will add the semicolon again.